server/database/schema.sql

```diff
@@ -1,46 +1,116 @@
-create table user (
-  id int primary key auto_increment not null,
-  username varchar(50) not null,
-  email varchar(100) not null unique,
-  password varchar(80) not null,
-  is_admin TINYINT not null default 0
-);
-
-create table team (
-  id int primary key auto_increment not null,
-  team_name varchar(255) not null
-);
-
-create table project (
-  id int primary key auto_increment not null,
-  name varchar(50) not null,
-  team_id int,
-  foreign key (team_id) references team(id)
-);
-
-create table task (
-  id int primary key auto_increment not null,
-  name varchar(50) not null,
-  description varchar(250),
-  is_archived TINYINT not null default 0,
-  project_id int not null,
-  team_id int not null,
-  foreign key (project_id) references project(id),
-  foreign key (team_id) references team(id)
-);
-
-create table user_has_team(
-  user_id int,
-  team_id int,
-  primary key (user_id, team_id),
-  foreign key (user_id) references user(id),
-  foreign key (team_id) references team(id)
-);
-
-create table user_has_task(
-  user_id int,
-  task_id int,
-  primary key (user_id, task_id),
-  foreign key (user_id) references user(id),
-  foreign key (task_id) references task(id)
-);
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Table `User`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `User` (
+  `id` INT NOT NULL,
+  `username` VARCHAR(50) NULL,
+  `Email` VARCHAR(100) NULL,
+  `password` VARCHAR(80) NULL,
+  `Is_admin` TINYINT NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `Team`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `Team` (
+  `id` INT NOT NULL,
+  `Team_Name` VARCHAR(55) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `Project`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `Project` (
+  `id` INT NOT NULL,
```

Comment:
all field names in lower case letter and check nullable property of fields - NOT NULL should be the majority